{
  "comments": [
    {
      "key": {
        "uuid": "12150160_1aabe6e5",
        "filename": "tests/datastore_access.cpp",
        "patchSetId": 1
      },
      "lineNbr": 360,
      "author": {
        "id": 1000000
      },
      "writtenOn": "2020-01-24T01:17:50Z",
      "side": 1,
      "message": "Vasek, do you think that this is sane? Does it look like a libnetconf2/libyang bug to you by any chance?",
      "range": {
        "startLine": 357,
        "startChar": 0,
        "endLine": 360,
        "endChar": 6
      },
      "revId": "88747fa522c0b281fdfbe0b007456e254f98acb2",
      "serverId": "e32f8df8-3db0-4f76-a6c6-fe9d6e69dd68",
      "unresolved": true
    }
  ]
}